federatedml/feature/OHE_with_alignment/OHE_alignment_host.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+#  added by jsweng
+#  alignemnet host
+
+import numpy as np
+
+from arch.api.utils import log_utils
+from federatedml.framework.homo.procedure import aggregator
+from federatedml.framework.homo.procedure import paillier_cipher
+from federatedml.feature.OHE_with_alignment.OHE_alignment_base import OHEAlignmentBase
+from federatedml.util import consts
+import functools
+import ast
+from collections import defaultdict
+
+
+LOGGER = log_utils.getLogger()
+
+
+class OHEAlignmentHost(OHEAlignmentBase):
```

Comment:
The logic of Host is almost same with Guest. Maybe you can create a "Client" class and make Guest and Host inherit it.

Reply:
done refactored and deleted this file